### PR TITLE
Automatically detect google app id in iOS

### PR DESCRIPTION
**Pull request by Cy-4AH**
Resolves https://github.com/fastlane/fastlane-plugin-firebase_app_distribution/issues/32

### CODE REVIEW

File: lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_helper.rb [hunk 1]
```diff
@@ -56,6 +56,14 @@ def cleanup_tempfiles
         return if @tempfiles.nil?
         @tempfiles.each(&:unlink)
       end
+
+      def findout_ios_app_id_from_archive(path)
+        appPath = `plutil -extract ApplicationProperties xml1 -o - "#{path}/Info.plist" | plutil -convert json -o - - | jq -r ".ApplicationPath"`.delete("\n")
```
Comment (snatchev) [inline lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_helper.rb]: fastlane has a dependency on the [plist gem](https://github.com/patsplat/plist). Please prefer that over shelling out to a binary if possible.

Comment (Cy-4AH) [reply lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_helper.rb]: Plist gem doesn't parse binary plists. In archive folder GoogleServices-Info.plist is already compiled to binary plist.

Comment (snatchev) [reply lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_helper.rb]: Ah, good catch! We also have the dependency on `CFPropertyList`

Comment (Cy-4AH) [reply lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_helper.rb]: @snatchev, thanks, I didn't know about it. I have migrated on `CFPropertyList`

**Comment by Cy-4AH** [comment]
@googlebot I signed it!

**Comment by Cy-4AH** [comment]
@snatchev, hello! I doesn't have merge button.

**Comment by snatchev** [comment]
@Cy-4AH thank you for the contribution (and ping)!

**Pull request merged**